### ISSUE TITLE
unit tests show yargs warning

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-cli#570]
**Description**
running `npm test` shows yargs warning:

```
Configuring yargs through package.json is deprecated and will be removed in the next major release, please use the JS API instead.
```

**Comment by tripodsan** [comment]
caused by configuration in mocha that causes yargs to warn. I created a mocha issue: https://github.com/mochajs/mocha/issues/3742

**Comment by tripodsan** [comment]
fixed with mocha 6.1.0 (5a02bfac102042757cfc340beedffa4bf43c30da)
